python/datasources/uhc.py

```diff
@@ -0,0 +1,122 @@
+import pandas as pd
+
+from ingestion.standardized_columns import Race
+import ingestion.standardized_columns as std_col
+
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util, constants
+
+UHC_RACE_GROUPS = [
+    'American Indian/Alaska Native',
+    'Asian',
+    'Black',
+    'Hawaiian/Pacific Islander',
+    'Hispanic',
+    'Multiracial',
+    'Other Race',
+    'White',
+    'All',
+]
+
+UHC_AGE_GROUPS = ['18-44', '45-64', '65+', 'All']
+
+UHC_SEX_GROUPS = ['Male', 'Female', 'All']
+
+UHC_RACE_GROUPS_TO_STANDARD = {
+    'American Indian/Alaska Native': Race.AIAN_NH.value,
+    'Asian': Race.ASIAN_NH.value,
+    'Black': Race.BLACK_NH.value,
+    'Hispanic': Race.HISP.value,
+    'Hawaiian/Pacific Islander': Race.NHPI_NH.value,
+    'Other Race': Race.OTHER_STANDARD_NH.value,
+    'White': Race.WHITE_NH.value,
+    'Multiracial': Race.MULTI_NH.value,
+    'All': Race.ALL.value,
```

Comment:
Rather than adding a new Race.ALL, why not just use Race.Total?

The convention so far has been to use "Total" everywhere in the BE and then we convert "Total" -> "All" on the FE (not a great system, but that's how it's been done so far). If the join with ACS population is planned to be done on the BE for this dataset at some point, you will have to use "Total" rather than "All" for it to work as things currently stand (for race, age, and sex)

Reply:
I was trying to sty ahead of the times https://github.com/SatcherInstitute/health-equity-tracker/issues/963

Reply:
Ah OK, in that case sgtm to leave what you have as-is :)